cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DeleteTest.scala

```diff
@@ -58,7 +58,9 @@ class DeleteTest extends DocumentingTest with QueryStatisticsTestSupport {
       }
     }
     section("Delete all nodes and relationships", "delete-delete-all-nodes-and-relationships") {
-      p("This query is not for deleting large amounts of data, but is useful when experimenting with small example data sets.")
+      warning(p(
```

Comment:
Apparently the build fails because `warning` is not found?

Reply:
Replacing with `important`.